main.py

```diff
@@ -0,0 +1,18 @@
+class Solution(object):
+    def twoSum(self, nums, target):
+        dicts = {}
+        for i in range(0, len(nums)):
+            dicts[nums[i]] = i
+        for i in range(0, len(nums)):
```

Comment:
```suggestion
        for i, num in enumerate(nums):
```

range всегда начинается с нуля, тут было бы достаточно `range(len(nums)):`

Reply:
О, точно, забыл!